### ISSUE TITLE
Show function source in documentation

### ISSUE THREAD

**Issue by Nodd** [spyder-ide/spyder#3185]
From #2980.
> 4) np.histog + Select histogram2d + Ctrl
> --> Right Side panel (Object Inspector or other), show Source Code file/ Source Code of numpy histogram2d + FolderName+Filename
> (it looks like your Goto function)

The original idea was to open the source file from the completion tooltip with a specific shortcut, but for the same reason as #3183 I think that this functionality belongs to the help menu.

**Comment by ccordoba12** [comment]
There is already the functionality to do this in the Help plugin, but I agree we need to make it more visible.

I agree this doesn't belong to the completion widget.

**Comment by jitseniesen** [comment]
This may probably not be the right place to ask, but what is the help plugin? Is it the same as the "Object inspector" pane? If so, then where is this functionality? I see that there is a "Show source" check box in the "Options" button, but that does not seem to do anything.

**Comment by ccordoba12** [comment]
> what is the help plugin?

Yep, it's the Object Inspector. It was renamed to *Help* for 3.0 :-)

> that does not seem to do anything.

It depends if there's something to show. What object are you trying to get its source code from?

**Comment by jitseniesen** [comment]
> It depends if there's something to show. What object are you trying to get its source code from?

`os.walk` (after importing `os`)

I investigated it a bit further, and found that it does work if it originates from Console, but not if it originates from Editor. In the latter case, clicking "Show source" does not do anything: it does not check the checkbox, it does not close the box (with two radiobuttons and two checkboxes; don't know the proper name) that opens when you click the Options button.

**Comment by ccordoba12** [comment]
It's true that this doesn't work when you select Editor as source. That's why it needs to be fixed :-)
